src/main/java/org/jitsi/impl/protocol/xmpp/colibri/ColibriConferenceImpl.java

```diff
@@ -280,6 +314,30 @@ public ColibriConferenceIQ createColibriChannels(
                         (ColibriConferenceIQ) response, contents);
 
         }
+        catch (Exception e)
+        {
+            try
+            {
+                synchronized (syncRoot)
```

Comment:
It's using `postEvent` method which just puts that event on a queue (single threaded) without blocking the current thread:
https://github.com/jitsi/jicoco/blob/master/src/main/java/org/jitsi/eventadmin/EventAdmin.java#L33